misc/scripts/install_d3d12_sdk_windows.py

```diff
@@ -14,21 +26,25 @@
     deps_folder = os.path.join("bin", "build_deps")
 
 # DirectX Shader Compiler
-dxc_version = "v1.7.2308"
-dxc_filename = "dxc_2023_08_14.zip"
+# Check for latest version: https://github.com/microsoft/DirectXShaderCompiler/releases/latest
+dxc_version = "v1.8.2403.2"
+dxc_filename = "dxc_2024_03_29.zip"
 dxc_archive = os.path.join(deps_folder, dxc_filename)
 dxc_folder = os.path.join(deps_folder, "dxc")
 # Mesa NIR
+# Check for latest version: https://github.com/godotengine/godot-nir-static/releases/latest
 mesa_version = "23.1.9"
 mesa_filename = "godot-nir-23.1.9.zip"
 mesa_archive = os.path.join(deps_folder, mesa_filename)
 mesa_folder = os.path.join(deps_folder, "mesa")
 # WinPixEventRuntime
-pix_version = "1.0.231030001"
+# Check for latest version: https://www.nuget.org/api/v2/package/WinPixEventRuntime (check downloaded filename)
+pix_version = "1.0.240308001"
 pix_archive = os.path.join(deps_folder, f"WinPixEventRuntime_{pix_version}.nupkg")
 pix_folder = os.path.join(deps_folder, "pix")
 # DirectX 12 Agility SDK
-agility_sdk_version = "1.610.4"
+# Check for latest version: https://www.nuget.org/api/v2/package/Microsoft.Direct3D.D3D12 (check downloaded filename)
+agility_sdk_version = "1.613.3"
```

Comment:
You'd also want to upgrade here (and regen docs):
https://github.com/godotengine/godot/blob/126a763f7ab9fa86ff8f32410f64bff2e603354a/core/config/project_settings.cpp#L1487

Reply:
Done. I've also updated the description in the class reference accordingly, and added a comment in the script as well.